### PR TITLE
Replace abandoned library jeremeamia/superclosure with recommended opis/closure

### PR DESCRIPTION
**Pull request by peteraba**
[jeremeamia/superclosure](https://github.com/jeremeamia/super_closure) is no longer maintained. the original author recommends [opis/closure](https://github.com/opis/closure).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "php": "^7.3 || ^8.0",
     "ext-openssl": "*",
     "ext-mbstring": "*",
-    "jeremeamia/superclosure": "~2.2",
+    "opis/closure": "~3.6",
```
Comment (davidbyoung) [inline composer.json:35]: We'll also need to update the composer.json in the [Routing directory](https://github.com/opulencephp/Opulence/blob/3710de51f0d3dbc4a2770aea40906a66348aaa7c/src/Opulence/Routing/composer.json)

**Review by davidbyoung — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (peteraba) [reply composer.json:35]: Interesting, I thought I grepped the code base afterwards :thinking:

**Comment by peteraba** [comment]
FYI, I have no further plans for Opulence for now, but will watch out for the `composer.json` files in the submodules from now on. I'm sure you had to fix them in a lot of places for the PHP 8.0 support.

**Review by davidbyoung — APPROVED** [review]

**Comment by davidbyoung** [comment]
Thanks!

**Pull request merged**